### ISSUE TITLE
Calling map on Ints

### ISSUE THREAD

**Issue by mfirry** [NFLabs/zeppelin#360]
I exectuted by mistake this pieace of code:
```3 map (_ + 2)```
and was surprised to discover this result:
```res83: Seq[org.apache.spark.sql.catalyst.expressions.Add] = ArrayBuffer((3 + 2))```

**Comment by swkimme** [comment]
What a funny issue! LOL
It's done by org.apache.spark.sql.catalyst.ImplicitOperators, because in Zeppelin SparkInterpreter.java state like intp.interpret("import sqlc._");
We need to check if Zeppelin need to import everything under sqlc.

**Comment by mfirry** [comment]
Yeah, funny... I was almost ashamed to report it.
Somehow it goes all the way up to ```org.apache.spark.sql.catalyst.trees.TreeNode``` if I'm not mistaken

**Comment by swkimme** [comment]
I guess @Leemoonsoo  wrote this code.
What if we change 
```
intp.interpret("import sqlc._");
 ```
into
```
intp.interpret("import sqlc.createSchemaRDD");
```
?

**Comment by swkimme** [comment]
Ah, my mistake. Another conversions are needed when we're writing Language-Integrated Relational Queries. 
Plz refer 
http://spark.apache.org/docs/1.1.0/sql-programming-guide.html
and search ```import sqlContext._ ```

What if we just close this issue?

**Comment by Leemoonsoo** [comment]
yes, if `import sqlc._` is not making serious side-effect, let's keep it.
